app/views/shared/_navbar.html.erb

```diff
@@ -1,9 +1,19 @@
 <nav class="bg-black h-28 w-screen flex flex-row ">
-  <div class="mx-4 py-5">
+  <div class="mx-4 py-5 flex items-center">
     <%= image_tag("logo.png", class:"my-0 h-4/5") %>
   </div>
-  <div class="mx-4 py-5">
-    <%= link_to "GetStarted", "events#index", class:"bg-black rounded py-5 px-2 text-white font-bold"%>
 
+  <div class="mx-4 flex items-center">
+    <%= print_home_link %>
+  </div>
+
+  <% if user_signed_in? %>
+    <div class="mx-4 flex items-center">
+      <%= print_get_started_link %>
+    </div>
+  <% end %>
+
+  <div class="mx-4 flex items-center">
+    <%= user_signed_in? ? print_sign_out_button : print_sign_in_link %>
   </div>
```

Comment:
It's better now but I think you can write it in a different form. I let you here an example.
I think that you can print the get started link when a user is singed in, otherwise just show the sign in link only

```
<div class="mx-4 flex items-center">
   <% if user_signed_in? %>
     <%= print_get_started_link %>
     <%= print_sign_out_button %>
  <% else %>
     <%= print_sign_in_link %>
  <% end %>
</div>
```

Reply:
Done!